_component_design/date-picker.md

```diff
@@ -0,0 +1,341 @@
+---
+title: Date Picker
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: Enable users to filter a list of events, or a report, by date ranges, or choose a single date for an event as they fill out a form.
+preview-image: preview-images/date-picker.svg
+status: unknown
```

Comment:
Is the status really still unknown?

Reply:
it is stable at this point.  will change.

Reply:
👍